modes/keyframes.js

```diff
@@ -148,15 +148,6 @@ module.exports = {
 		let [lastWidth, lastHeight] = [keyFrames[lastKf].width, keyFrames[lastKf].height]
 		let [nextWidth, nextHeight] = lastKf === keyFrames.length - 1 ? [lastWidth, lastHeight] : [keyFrames[lastKf + 1].width, keyFrames[lastKf + 1].height]
 
-		if (keyFrames[lastKf + 1].interpolation.toLowerCase() === 'instant') {
```

Comment:
What's this for?

Reply:
this was originally me mentally alternating between having `instant` mode be about interpolating *to* or *from* it, but apparently i accidentally did both when it was meant to only be when interpolating *from* instant to something else - this fixes that oversight

Reply:
i just happen not to have caught it before, apparently